tests/bluetooth/audio/ascs/mocks/ascs_help_utils.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/gatt.h>
+
+#define ASCS_ASE_STATE_IDLE                     0x00
+#define ASCS_ASE_STATE_CODEC_CONFIGURED         0x01
+#define ASCS_ASE_STATE_QOS_CONFIGURED           0x02
+#define ASCS_ASE_STATE_ENABLING                 0x03
+#define ASCS_ASE_STATE_STREAMING                0x04
+#define ASCS_ASE_STATE_DISABLING                0x05
+#define ASCS_ASE_STATE_RELEASING                0x06
```

Comment:
We have `enum bt_audio_state` in `audio.h`

Reply:
removed